spec_mode/mirimrs/notebook1/MRS_Notebook1.ipynb

```diff
@@ -24,7 +24,7 @@
    "source": [
```

Comment:
<p>This is an example of a cell with several PEP8 warnings printed as output below the cell. (The <code>IndexError</code> is a separate issue that should be fixed once the notebook runs in full.) In this case, every warning is for a location where there's no space after a comma -- index 56 of line 11, index 14 of line 10, etc. Remember that the preferred style is <code>a, b, c</code> instead of <code>a,b,c</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>